src/dsp_tools/commands/xmlupload/check_consistency_with_ontology.py

```diff
@@ -0,0 +1,123 @@
+# sourcery skip: use-fstring-for-concatenation
+import regex
+from lxml import etree
+from regex import Pattern
+
+from dsp_tools.commands.xmlupload.models.ontology_diagnose_models import InvalidOntologyElements, OntoCheckInformation
+from dsp_tools.commands.xmlupload.ontology_client import OntologyClientLive
+from dsp_tools.models.exceptions import BaseError
+
+defaultOntologyColon: Pattern[str] = regex.compile(r"^:\w+$")
+knoraUndeclared: Pattern[str] = regex.compile(r"^\w+$")
+genericPrefixedOntology: Pattern[str] = regex.compile(r"^[A-Za-z]+-?[A-Za-z]+:\w+$")
```

Comment:
Are you sure ontos can contain only 1 single dash? I'd feel more comfortable with:

```suggestion
genericPrefixedOntology: Pattern[str] = regex.compile(r"^[\w\-]+:\w+$")
```